alert-policies/ansible-automation-controller/TimeSpentSavingEventstoDatabase.yml

```diff
@@ -0,0 +1,37 @@
+name: Time Spent Saving Events to Database
+
+description: |+
+  This alert is triggered when the time spent saving events to database is higher than 6 during 5 minutes.
```

Comment:
```suggestion
  This alert is triggered when the time spent saving events to database is higher than 6 within 5 minutes.
```

Reply:
Hi @**paperclypse** ,
Reviewed text correction has been changed. Kindly review it.